data/mlox_base.txt

```diff
@@ -90,6 +90,8 @@ Clean UMPP-fixes-BTB-v2.esp
 Clean UMPP-fixes-v2.esp
 ;;Midgetalien's Bloodmoon Patch 0.5.1 [Midgetalien], q.v.
 Clean_bloodmoonpatch_05.esp
+;;General Fixes Mod by Fritz
+GFM_6.3.2_1C.esp

```

Comment:
Is correcting official fails in any term. Should be loaded instantly behind official dlc's! No exclusion.